tools/add_note.rb

```diff
@@ -35,29 +35,20 @@ def parse_file(file)
   json_contents
 end
 
+#Deep copy is unnecessary
 def get_last_guid(file_json)
-  Marshal.load(Marshal.dump(file_json['notes'].last['guid']))
+  file_json['notes'].last['guid'].dup
```

Comment:
Wasn't this already merged in?

Reply:
Yes, it was. I am unfamiliar with the process (if one exists) of excluding commits from pull requests.